includes/customize-feed.php

```diff
@@ -256,15 +262,13 @@ function feed_item() {
 add_action( 'rss2_item', __NAMESPACE__ . '\feed_item' );
 
 /**
- * Adjust the enclosure feed for podcasts.
+ * Displays the enclosure feed for podcasts.
  *
- * @param  string $enclosure The enclosure (media url).
+ * @param  int $post The post ID.
```

Comment:
Please update the type hint. When called from customize-feed.php:241, the `$post` argument is of type `WP_Post`, and also within this function, `$post` is considered as post object. Probably also needs to typecast with `$post = get_post( $post );` at the top of this function.